frontend/package.json

```diff
@@ -58,7 +58,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "check-ci": "CI=true eslint src && prettier --config ./.prettierrc --check 'src/**/*.js'",
-    "format": "CI=true eslint src --fix && prettier --config ./.prettierrc --check 'src/**/*.js' --write"
+    "format": "CI=true eslint src --fix && prettier --config ./.prettierrc 'src/**/*.js' --write"
```

Comment:
key change

Reply:
Why remove `--check`?

Reply:
Reverted...turned out I had the wrong version of prettier.